chirp/drivers/uv5r.py

```diff
@@ -359,6 +357,19 @@ def _upper_band_from_image(radio):
     return _upper_band_from_data(radio.get_mmap())
 
 
+def _read_from_data(data, data_start, data_stop):
+    data = data[data_start:data_stop]
+    return data
+
+
+def _get_data_from_image(radio, _data_start, _data_stop):
+    image_data = _read_from_data(
+        radio.get_mmap().get_byte_compatible(),
```

Comment:
This `get_byte_compatible()` should be a no-op on anything that is byte-clean. I assume this is just copy pasta and not intentional?

Reply:
Thanks. I will look at it after supper.